### PR TITLE
docs: [vision-on-sample-app][11/n] Identify

### PR DESCRIPTION
**Pull request by Ahmed-CIO**
This is a PR in a series of PRs to build sample app that runs Identify and Track in VisionPro
The app is built to be an interactive walk through guidance for developers who are integrating Customer.io Swift SDK in their VisioPro apps.
For more context:

- See the project [one pager](https://www.notion.so/custio/Spacial-Identify-and-Tracking-6587d848bbde494095c0585237326ed3?pvs=4)
- See [Linear Issue](https://linear.app/customerio/issue/MBL-115/sample-app-for-visionpro)

This PR introduces a the "Identify" tutorial screen. It demonstrates the requirements of the API, allows the user to enter profile attributes to be send alongside the identify call.
Once the user hits the "Identify" button:
- The app will navigate to the "how to test identify" screen that's still blank.
- The identify details will be saved in the AppState and used to call idenfity
- Next app run will use the persisted data to call idenfity on app start

**UI:**
- Add/remove profile attributes and make sure they reflect in the code snippet

**Persistence**
- Ensure the "Identify" button indeed calls `CustomerIO.shared.identify`
- Ensure next app start does call identify after initializing the SDK

https://github.com/customerio/customerio-ios/assets/142905621/be12ca03-a6cc-4d84-8ed2-19fa5b35f0a4

---

**Stack**:
- #628
- #627 ⬅
- #626 
- #625 
- #624
- #623
- #622
- #621
- #620
- #619
- #618

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

File: Apps/VisionOS/VisionOS/AppDelegate.swift [hunk 1]
```diff
@@ -30,6 +30,15 @@ class AppDelegate: NSObject, UIApplicationDelegate {
                 config.logLevel = .debug
             }
         }
+        
+        // If user already logged in, let's
+        // call identify
+        let profile = AppState.shared.profile
+        if profile.loggedIn {
+            CustomerIO.shared.identify(
```
Comment (ami-aman) [inline Apps/VisionOS/VisionOS/AppDelegate.swift:38]: Wondering why is `identified` being called again? 🤔 
Our SDK retains the last logged in user id. Do you have a use case you want to cover with this ?

File: Apps/VisionOS/VisionOS/TutorialViews/IdentifyTutorialView.swift [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    let onSuccess: (Profile) -> Void
+
+    var body: some View {
+        ScrollView(.vertical) {
+            VStack(alignment: .leading) {
+                Markdown {
+"""
+In CustomerIO many things revolve around People. What actions
+(tracked events) they do, what we know about them (their attributes),
+what devices they use and what are the capabilities of thess devices
```
Comment (ami-aman) [inline Apps/VisionOS/VisionOS/TutorialViews/IdentifyTutorialView.swift:42]:
```suggestion
what devices they use and what are the capabilities of these devices
```

File: Apps/VisionOS/VisionOS/AppDelegate.swift (hunk 1 above)
Comment (Ahmed-CIO) [reply Apps/VisionOS/VisionOS/AppDelegate.swift:38]: Good call out, I didn't know that!
Will remove this call

File: Apps/VisionOS/VisionOS/ViewUtils/CodeSnippetHelper.swift [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    var sep = ""
+    senatizedProperties.forEach { p in
+        res += """
+        \(sep)"\(p.name)": "\(p.value)"
+        """
+        sep = ", "
+    }
+    res += "]"
+    return res
+}
```
Comment (levibostian) [inline Apps/VisionOS/VisionOS/ViewUtils/CodeSnippetHelper.swift:36]: As written, this function took a lot of time to review to understand what it's doing. I believe, it's doing quite a lot. 

Could this be refactored with the goal of making it faster and easier to read? I think the function body is doing multiple things which is part of what makes it complex. Splitting this up into multiple function calls could help? 

At a minimum, adding comments explaining a chunk of code could help. I am open to ideas.

**Review by levibostian — APPROVED** [review]

**Pull request closed without merging**